database/sqlite/sqlite_aclk.c

```diff
@@ -258,6 +258,46 @@ void aclk_sync_exit_all()
     uv_mutex_unlock(&aclk_async_lock);
 }
 
+static int create_host_callback(void *data, int argc, char **argv, char **column)
+{
+    UNUSED(data);
+    UNUSED(argc);
+    UNUSED(column);
+
+    char guid[UUID_STR_LEN];
+    uuid_unparse_lower(*(uuid_t *)argv[0], guid);
+
+    struct rrdhost_system_info *system_info = callocz(1, sizeof(struct rrdhost_system_info));
+    system_info->hops = str2i((const char *) argv[7]);
+
+    RRDHOST *host = rrdhost_find_or_create(
+          (const char *) argv[1]
+        , (const char *) argv[2]
+        , guid
+        , (const char *) argv[4]                                    //os
+        , (const char *) argv[5]                                    // timezone
+        , (const char *) argv[9]                                    // abbrev timezone
+        , argv[10] ? str2uint32_t(argv[10]) : 0
+        , (const char *) argv[6]                                    // tags
+        , (const char *) (argv[11] ? argv[11] : "unknown")          // program name
+        , (const char *) (argv[12] ? argv[12] : "unknown")          // program version
+        , argv[3] ? str2i(argv[3]) : 1                              // update every
+        , argv[13] ? str2i(argv[13]) : 0                            // entries
```

Comment:
It would be better to use an enum instead of magic numbers. It's easier to follow a list of words than count fields if anything changes in the `sqlite3_exec` call.

Reply:
Added